docs/glossary.md

```diff
@@ -0,0 +1,79 @@
+---
+# Glossary position should be as last as possible, but before the Resources category.
+sidebar_position: 999
+---
+
+# Glossary
+
+Use the glossary to find definitions and further information for new or
+ambiguous terms.
+
+### Aggregator
+
+A server that implements the Distributed Aggregation Protocol (DAP). Divvi Up
+and its partners operate DAP-compliant aggregators. Aggregators can be
+self-hosted.
+
+Aggregators ingest measurements sent from clients. They perform aggregation of
+batches or reports into aggregations, and service [collector](#collector)
+requests.
+
+Aggregators can be in one of two roles, [Leader](#leader) or [Helper](#helper).
+
+Examples of aggregator software include [Janus (ISRG)][janus], and [Daphne
+(Cloudflare, helper-only)][daphne].
+
+### Client
+
+An end-user application whose developer wishes to collect metrics on.
+
+### Collector
+
+An application that collects aggregate shares from aggregators and combines them
+into the final aggregate. This is most often operated by the application
+developer.
+
+A library suitable for building collectors is found in the crate
+[`janus_collector`][janus_collector]. A primitive collector implementation is
+available in [Janus][collect.rs].
+
+### Helper
+
+A DAP aggregator that performs aggregation and collection in response to
+direction from the Leader. The system requirements for a helper are lower than a
+leader.
+
+### Leader
+
+A DAP aggregator that directly ingests metrics, creates aggregation and
+collection jobs, and drives those jobs to completion. The system and network
+bandwidth requirements for a leader are higher than for a helper.
+
+### Query type
+
+Describes how a batch of reports are to be aggregated. There are two query types
+defined by DAP:
+
+- Time-interval: Batches are queried based on some time range.
+- Fixed-size: Batches contain a fixed number of reports. The collector queries
```

Comment:
Given that we have a max_batch_size and min_batch_size, it's not strictly true that each batch would contain the same number of reports--this might mislead someone implementing a collector.